### ISSUE TITLE
Registration page

### ISSUE THREAD

**Issue by JuliannaR** [Quixote-project/concierge#42]
message.gccollab.ca pilot platform: The registration page currently loses focus order, navigating via keyboard is compromised and would immediately detract some users with disabilities. This is failure WCAG 2.0.

**Comment by bartjkdp** [comment]
Thanks a lot for reporting this! I just submitted a fix for this in fcf35a1. You can test it out on Pleio account (we are following the latest version), https://account.pleio.nl/register/. But it will probably be deployed at GCTools as well in a couple of weeks.
